packages/shylu/shylu_dd/frosch/src/InterfaceSets/FROSch_InterfaceEntity_def.hpp

```diff
@@ -312,7 +312,7 @@ namespace FROSch {
                 case ConstantDistanceFunction:
                     for (UN i=0; i<NodeVector_.size(); i++) {
                         for (UN j=0; j<CoarseNodes_->getNumEntities(); j++) {
-                            DistancesVector_[i][j] = 1.0;
+                            DistancesVector_[i][j] = Teuchos::ScalarTraits<SC>::one();
```

Comment:
I'm not sure what `DistancesVector_` is, but consider a way to avoid writing for loops here.  Maybe that object has a "set all to value" or "fill" method.

Reply:
@mhoemmen At the moment, this is an `ArrayRCP`. At least I haven't found a method which does this. I could probably also use a MultiVector object instead. I will put this on my todo list as well.